skrf/calibration/calibration.py

```diff
@@ -2433,10 +2439,21 @@ def __init__(self, measured, ideals=None, estimate_line=False,
             ideal_thru.s[:,0,1] = 1
             ideals[0] = ideal_thru
 
+        orig_ideal_thru = None
+
+        if npy.any(ideals[0].s[:,1,0] != 1) or npy.any(ideals[0].s[:,0,1] != 1):
+            # Don't modify the original network
+            orig_ideal_thru = ideals[0].copy()
```

Comment:
```suggestion
            orig_ideal_thru = ideals[0]
```
This copy is not required